src/scripts.js

```diff
@@ -4,25 +4,23 @@ import './css/styles.scss';
 import recipeData from './data/recipes';
 import ingredientData from './data/ingredients';
 import users from './data/users';
-
 import Pantry from './pantry';
 import Recipe from './recipe';
 import User from './user';
 import Cookbook from './cookbook';
+import domUpdates from './domUpdates';
```

Comment:
Thanks for catching that we needed a separate domUpdates file. Would love to hear more about how you decided what should go into that file vs. the regular scripts file in our work session tomorrow.